Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_11_0.md

```diff
@@ -0,0 +1,43 @@
+
+#### Integrations
+- **Microsoft Defender for Endpoint**
+- Set Fetch incident to be true by default.
+- Set Default mapper to be **Microsoft Defender For Endpoint Mapper**
+
+#### Mappers
+- **Microsoft Defender For Endpoint Mapper**
+- Added new fields to be associated with new malware type
+
+#### Playbooks
```

Comment:
all 6.2 to 6.5

Reply:
I'm not sure what this means - should 6.0 be changed to 6.2 and 6.2 changed to 6.5? I will leave this for @ssokolovich to update

Reply:
It means that all the new PBs must be 6.5. Checking that.